### DISCUSSION TITLE
Do I need to create user if I am using SMTP_ONLY?

### DISCUSSION THREAD

**Discussion by winston0410** [docker-mailserver/docker-mailserver#2774] [Q&A] [answered]
I guess it is a newbie question. I am trying to use this image with SMTP_ONLY for sending email, do I need to create user with `./setup.sh email add`? I will be connecting with Postfix later on with smtp url.

**Comment by casperklein** [comment, marked as answer]
No user is needed. However I noticed a bug, that prevents the correct startup of DMS when no user is configured and no other config files are present. For now, you can add a dummy user.

I'll provide a fix soon.

Edit: Will be fixed with merge of https://github.com/docker-mailserver/docker-mailserver/pull/2776

**Reply by winston0410** [reply]
thats perfect thanks!
